README.md

```diff
@@ -45,5 +45,6 @@ POSTGRES_PASSWORD=changeIt
 POSTGRES_DB=testdb
 POSTGRES_PORT=5432
 POSTGRES_HOST=localhost
-SCHEDULE_CRONEXPRESSION=0 0 0 * * Mon
+UPDATE_CENTER_CRON=0 0 0 * * Mon
+PROBE_ENGINE_CRON=0 30 0 * * *
```

Comment:
I know this is just the example of cron expressions used for probeEngine scheduling and updateCenter scheduling, but which one would be more frequent?

In this case, we're running the probe engine more frequently than the update center which (correct me if I'm wrong) wouldn't make sense. This is a general question, not dependent on this specific example.

Reply:
> I know this is just the example of cron expressions used for probeEngine scheduling and updateCenter scheduling, but which one would be more frequent?

the `PROBE_ENGINE_CRON` is saying "everyday at 30 past midnight.
the `UPDATE_CENTER_CRON` is saying "every monday at midnight.